src/Dynatrace.OpenTelemetry.Exporter.Metrics/DynatraceMetricsMapper.cs

```diff
@@ -0,0 +1,114 @@
+// <copyright company="Dynatrace LLC">
+// Copyright 2021 Dynatrace LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Collections.Generic;
+using OpenTelemetry.Metrics.Export;
+using DynatraceMetric = Dynatrace.MetricUtils.Metric;
+using DynatraceMetricFactory = Dynatrace.MetricUtils.MetricsFactory;
+using DynatraceMetricException = Dynatrace.MetricUtils.MetricException;
+using System.Text;
+using Microsoft.Extensions.Logging;
+
+internal static class DynatraceMetricsMapper
+{
+
+	/// <summary>
+	/// Combines metric namespace and key into a single key for use in <see cref="Dynatrace.MetricUtils.Metric">.
+	/// </summary>
+	/// <returns>A metric key in the form {MetricNamespace}.{MetricKey}</returns>
+	private static string CreateMetricKey(Metric metric)
+	{
+		var keyBuilder = new StringBuilder();
+		if (!string.IsNullOrEmpty(metric.MetricNamespace))
+		{
+			keyBuilder.Append($"{metric.MetricNamespace}.");
+		}
+		keyBuilder.Append(metric.MetricName);
+		return keyBuilder.ToString();
+	}
+
+	internal static IEnumerable<DynatraceMetric> ToDynatraceMetric(Metric metric, ILogger logger)
+	{
+		var metricName = CreateMetricKey(metric);
+		foreach (var metricData in metric.Data)
+		{
+			DynatraceMetric dynatraceMetric = null;
+			var timestamp = metricData.Timestamp;
```

Comment:
Does it make sense to create these two vars here since it's already in the `metricData`?

Reply:
You're right, not really. Removed them in  and used `metricData.Timestamp` and `metricData.Labels` instead in [865ec77](https://github.com/dynatrace-oss/opentelemetry-metric-dotnet/pull/24/commits/865ec7702b5927bafbac1a70801480ed06bbbd30)